src/components/ResultsList.tsx

```diff
@@ -35,7 +31,7 @@ const ResultsList = ({ title, results, navigation }) => {
         }}
       />
     </View>
-  );
+  ) : null;
```

Comment:
It is not in the scope of this task, and you probably already know it, but just in case: in production there should always a view dedicated to let the user know there are no results.